src/test/groovy/ee/tuleva/onboarding/error/ErrorHandlingControllerAdviceSpecification.groovy

```diff
@@ -0,0 +1,104 @@
+package ee.tuleva.onboarding.error
+
+import ee.tuleva.onboarding.auth.response.AuthNotCompleteException
+import ee.tuleva.onboarding.error.exception.ErrorsResponseException
+import org.springframework.validation.DirectFieldBindingResult
+import spock.lang.Specification
+import org.springframework.http.HttpStatus
+import ee.tuleva.onboarding.error.response.ErrorResponseEntityFactory
+import ee.tuleva.onboarding.error.response.ErrorsResponse
+import org.springframework.http.ResponseEntity
+import ee.tuleva.onboarding.mandate.exception.IdSessionException
+import io.jsonwebtoken.ExpiredJwtException
+import ee.tuleva.onboarding.auth.ExpiredRefreshJwtException
+
+class ErrorHandlingControllerAdviceSpecification extends Specification {
```

Comment:
Should be called just "...Spec"

Reply:
There is also ErrorHandlingControllerSpec, with a different test setup. I did not want to delete it